.circleci/config.yml

```diff
@@ -299,13 +297,30 @@ jobs:
             - v1-bazel-cache-core-{{ .Branch }}
             - v1-bazel-cache-core-main
 
+      # union the bundle targets until //... doesnt explode analyzing jvm/android targets
```

Comment:
@brocollie08 are you planning on tackling coverage with you bazel enhancments ticket, or should we make a new issue?

Reply:
new issue, I'll create that too